staging/src/k8s.io/api/core/v1/toleration_test.go

```diff
@@ -114,6 +114,20 @@ func TestTolerationToleratesTaint(t *testing.T) {
 			},
 			expectTolerated: false,
 		},
+		{
+			description: "toleration have an empty key with operator Exists, expect tolerate everything",
```

Comment:
s/have an empty key with operator Exists/have an empty Key and empty Effect with operator Exists/

Reply:
Done